orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/AbstractInstancesCheckTask.groovy

```diff
@@ -119,7 +126,16 @@ abstract class AbstractInstancesCheckTask extends AbstractCloudProviderAwareTask
         }
       }
 
-      verifyServerGroupsExist(stage)
+      try {
+        verifyServerGroupsExist(stage)
+      } catch (MissingServerGroupException e) {
+        if (waitForUpServerGroup()) {
```

Comment:
Seems reasonable ... it's probably overkill to wait for up to 2hrs for a server group to exist.

I'd suggest short circuiting to failure after 5 or 10 minutes of waiting.